pallets/liquid-staking-v2/src/lib.rs

```diff
@@ -136,11 +282,38 @@ mod pallet {
     #[pallet::getter(fn exchange_rate)]
     pub type ExchangeRate<T: Config> = StorageValue<_, Rate, ValueQuery>;
 
-    /// Total amount of staked assets in relaycahin.
+    /// The total amount of a staking asset.
     #[pallet::storage]
     #[pallet::getter(fn staking_pool)]
     pub type StakingPool<T: Config> = StorageValue<_, BalanceOf<T>, ValueQuery>;
 
+    /// Records stake detail during each era.
+    #[pallet::storage]
+    #[pallet::getter(fn account_stake)]
+    pub type AccountStake<T: Config> = StorageDoubleMap<
+        _,
+        Twox64Concat,
+        EraIndex,
+        Twox64Concat,
+        T::AccountId,
+        StakeMisc,
+        ValueQuery,
+    >;
+
+    /// Record all the pending unstaking requests.
+    /// Key is the owner of assets.
+    #[pallet::storage]
+    #[pallet::getter(fn account_unstake)]
+    pub type AccountUnstake<T: Config> = StorageDoubleMap<
+        _,
+        Twox64Concat,
```

Comment:
same as above

Reply:
addressed by using Blake2_128Concat for EraIndex.